src/solvers/fd_tree/fd_2d.jl

```diff
@@ -0,0 +1,199 @@
+
```

Comment:
OK! I guess there's no way around this meaningless acronym, since I have already accepted the use of DGSEM 😄 But it indicates that it has to do with FD methods, which most people will recognize, but also that it is somewhat special, so I guess not that bad overall.

Reply:
And it's consistent with the length of DGSEM :wink:

Reply:
I am not sure if I like the fact that you consistently consistency-snipe me.... 😠